### PR TITLE
Added a new quote

### PR DESCRIPTION
**Pull request by elenakout**
New quote form novelist Zadie Smith

## ✅ New Quote Checklist
- [ x ] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [ x ] The quote(s) added are about **learning a foreign language**.
- [ x ] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [ x ] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

**Comment by Kvaibhav01** [comment]
@elenakout please add quotes relate to "learning a foreign language" as suggested in [contribution guidelines](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_quote.md#-checklist).

File: server/data/quotes.json [hunk 1]
```diff
@@ -570,5 +570,9 @@
   {
     "author": "William Shakespeare",
     "sentence": "My words fly up, my thoughts remain below. Words without thoughts never to heaven go."
-  }
+  },
+  {
+    "author": "Zadie Smith",
+    "sentence": "The past is always tense, the future perfect."
+  },
```
Comment (jayehernandez) [inline server/data/quotes.json:577]: Also apart from @Kvaibhav01’s comment, please remove the extra comma here as it is an invalid JSON

**Review by jayehernandez — CHANGES_REQUESTED** [review]

File: server/data/quotes.json (hunk 1 above)
Comment (elenakout) [reply server/data/quotes.json]: I think I managed to correct the pr

**Review by jayehernandez — APPROVED** [review]
Thank you for contributing to Letra ☀️, @elenakout!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**